src/client/components/Subheader/index.js

```diff
@@ -1,18 +1,42 @@
 import BPOButton from "../BPOButton";
+import Dropdown from "react-bootstrap/Dropdown";
 import React from "react";
+import i18n from "../../i18n";
 import { useTranslation } from "react-i18next";
 
 function Subheader() {
   const { t } = useTranslation();
 
+  const changeLanguage = (lng) => {
+    i18n.changeLanguage(lng);
```

Comment:
Does this also update the URL to include the param? I'm not sure if it should. If it does, we need to also update all the routes.

Alternately, let's not update the URL (so we can deploy the switcher) and we can update it in a separate change (probably part of issue #204).